src/sbt-test/sbt-web/asset-pipeline/build.sbt

```diff
@@ -31,8 +31,16 @@ jsmin := {
     val minFile = targetDir / "js" / "all.min.js"
     IO.touch(minFile)
     val minMappings = Seq(minFile) pair Path.relativeTo(targetDir)
-    minMappings ++ other
+    val convertedMinMappings = minMappings.map { case (file, path) =>
+      SbtWeb.asFileRef(file, fileConverter.value) -> path
+    }
+    convertedMinMappings ++ other
   }
 }
 
 pipelineStages := Seq(jsmin)
+
+TaskKey[Unit]("fileCheck") := {
+  assert((target.value / "jsmin" / "public" / "js" / "all.min.js").exists())
```

Comment:
The original check was `$ exists target/web/stage/js/all.min.js`, so we should keep that? (I ran that scripted test locally, both file will be generated anyway, but maybe better to keep things how they were before?)

Reply:
Good find, thanks! Fixed.